### PR TITLE
#415: fix image uploading in tests

### PR DESCRIPTION
**Pull request by vladarefiev**
PR for #415:
- created image fixture
- fixed tests with uploading images using multipart

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#461](https://github.com/timelesslounge/timelessis/issues/461) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#461](https://github.com/timelesslounge/timelessis/issues/461) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/ce23df76-373d-4085-ae2b-b47768796747); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/restaurants/table_shapes/forms.py [hunk 1]
```diff
@@ -9,17 +9,8 @@
 class TableShapeForm(forms.ModelForm):
     """Base form for table shape"""
 
-    picture = FileField(
-        "Table Shape Image",
-        validators=[FileRequired(), FileAllowed(IMAGES, "Images only!")])
-
```
Comment (amihaiemil) [inline timeless/restaurants/table_shapes/forms.py]: @vladarefiev I don't really understand, why did you remove the ``picture`` field and what do we have instead of it? :D

**Comment by amihaiemil** [comment]
@vladarefiev and there is a conflict now :D

File: timeless/restaurants/table_shapes/forms.py (hunk 1 above)
Comment (vladarefiev) [reply timeless/restaurants/table_shapes/forms.py]: @amihaiemil I have no idea why it's was overridden,. it was mistake ;)

**Comment by vladarefiev** [comment]
@amihaiemil Oops, fixed :D

**Comment by amihaiemil** [comment]
@vladarefiev I don't see any changes with your latest commit :D

**Comment by vladarefiev** [comment]
@amihaiemil Sorry, fixed now ;)

**Comment by amihaiemil** [comment]
@vladarefiev now ``test_edit`` is failing :(

**Comment by vladarefiev** [comment]
@amihaiemil Forgot to commit skipping :D

**Comment by vladarefiev** [comment]
@amihaiemil fixed. now `test_edit` is skipped, because uploading pictures works, puzzle for it in `test_edit` had already been provided

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 26 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#461](https://github.com/timelesslounge/timelessis/issues/461) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@amihaiemil According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by amihaiemil** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@amihaiemil thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)
